tendrl/node_agent/gluster_integration/flows/import_cluster.py

```diff
@@ -24,9 +27,13 @@ def run(self):
                                            json.dumps(job))
         if self.node_id in node_list:
             self.parameters['fqdn'] = socket.getfqdn()
-            gluster = "git+https://github.com/Tendrl/gluster_integration.git" \
-                      "@v1.0"
-            self.parameters['Package.name'] = gluster
+            if config.get("node_agent", "deployment_type") == "production":
```

Comment:
Same as in the ceph_integration case -- turn this into YAML and supply it as a common library method that allows package installation or git checkouts for provisioning components. The YAML flow would then invoke the library method with the appropriate parameters. These parameters (package names, repo paths, commands to be executed etc.) should be hardcoded in individual flows' YAML files.